runtime/common/src/migrations.rs

```diff
@@ -145,20 +150,7 @@ where
     /// Run a standard pre-runtime test. This works the same way as in a normal runtime upgrade.
     #[cfg(feature = "try-runtime")]
     fn pre_upgrade(&self) -> Result<Vec<u8>, sp_runtime::DispatchError> {
-        const CONFIGURATION_ACTIVE_CONFIG_KEY: &[u8] =
-            &hex_literal::hex!("06de3d8a54d27e44a9d5ce189618f22db4b49d95320d9021994c850f25b8e385");
-
-        let old_config_bytes =
-            frame_support::storage::unhashed::get_raw(CONFIGURATION_ACTIVE_CONFIG_KEY)
-                .expect("configuration.activeConfig should have value");
-        // This works because there is no enum in the v2
-        assert_eq!(
-            old_config_bytes.len(),
-            HostConfigurationV2::default().encoded_size()
-        );
-
-        use parity_scale_codec::Encode;
-        Ok((old_config_bytes).encode())
+        Ok(vec![])
```

Comment:
let's try to do the pre and post upgrade hooks no?

Reply:
Not sure what to do here

Reply:
easy: you take whatever you have before the migration and you check in the post hook that the migration has completed as you expected

Example: you output the old_config_bytes, which dont include your new rotationmethod

then in the post hook you recive that, you decode it accordingly, and then you read the value after the migration. If everything goes well,the post migration hook should contain the same values that you output in the pre but with the addition of the new rotation values

Reply:
Added this to the pre_upgrade, I believe it's enough:

```rust
        let old_config: Result<HostConfigurationV3, _> =
            DecodeAll::decode_all(&mut old_config_bytes.as_ref());
        let new_config: Result<HostConfiguration, _> =
            DecodeAll::decode_all(&mut old_config_bytes.as_ref());

        assert!(old_config.is_ok());
        assert!(new_config.is_err());
```

Reply:
no with this you are only checking that you cannot decode with the new config before the runtime-upgrade haha. At least add the opposite in the post upgrade

Reply:
*fixed*